### PR TITLE
DeploymentsList: Replace "least healthy" with "most active" deployments

### PR DESCRIPTION
**Pull request by rmars**
- Replace the Least Healthy Deployments section at the top of the deployments list page with most active deployments, sort by RPS
- Fix a bug where the titles of these tiles weren't being displayed because of an old usage of ConduitLink

screen shot 2018-02-05 at 2 52 39 pm

### CODE REVIEW

**Comment by briansmith** [comment]
Why would the user care about the most active deployments instead of the least healthy ones? I understand that "least healthy" sounds bad but at the same time I think that's a very useful feature if it works. Are we removing it because it doesn't work or just because it sounds bad?

**Comment by rmars** [comment]
It works fine.  We could also do one row of "most active" and one row of "least healthy". @wmorgan thoughts?

**Comment by adleong** [comment]
My 0.02$: In the absence of a well thought out story for what this is for, I don't think either "most active" or "least healthy" is particularly useful.  I'd be inclined to remove the summary graph row altogether.

**Comment by wmorgan** [comment]
This is not my favorite forum for justifying product decisions, but here we go. A very common NUX is when all services are at 100% SR. In that case, this section is poorly named and uninformative. We could have these graphs only appear when there are services < 100% SR, but opens up a whole set of design questions around what to do when SR numbers change, what about >3 low-SR services, etc.

For now, it is better to have something consistent and at least interesting. With sortable tables, it is very easy to find low-SR services anyways.

**Comment by briansmith** [comment]
> My 0.02$: In the absence of a well thought out story for what this is for, I don't think either "most active" or "least healthy" is particularly useful. I'd be inclined to remove the summary graph row altogether.

I agree with this. It doesn't seem like we have a great justification for these features, esp. based on what William said. Better just scrap them both and focus on more core functionality.

**Comment by wmorgan** [comment]
Let's continue with this ticket for now, please (it is a small amount of work), and address this sort of feedback in a future design review.

File: web/app/js/components/DeploymentSummary.jsx [hunk 1]
```diff
@@ -8,8 +8,9 @@ export default class DeploymentSummary extends React.Component {
       return this.props.data.name;
     } else {
       return (<this.props.api.ConduitLink
-        to={`/deployment?deploy=${this.props.data.name}`}
-        name={this.props.data.name} />);
+        to={`/deployment?deploy=${this.props.data.name}`}>
+        {this.props.data.name}
+      </this.props.api.ConduitLink>);
```
Comment (klingerf) [inline web/app/js/components/DeploymentSummary.jsx]: Nice catch :)

**Pull request merged**